Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo

```diff
@@ -320,9 +320,16 @@ load inertia.
 <p>
 This example animates a motion of a detailed model of the robot
 with predefined axes' angles over time.
-For animation CAD data is used.
+For animation, CAD data is used.
 Translate and simulate with the default settings
-(default simulation time = 3 s).
+(default simulation stop time = 2&nbsp;s).
+<p>
+</p>
+In the path planning block, there is incorporated a simulation termination condition.
+Thus, the simulation can be terminated before reaching the stop time.
+The condition depends on the joints' start and end positions, reference speeds
+and reference accelerations.
+For current settings, the simulation should be terminated at about 1.85&nbsp;s.
```

Comment:
I'd rather skip this last sentence.

Reply:
I'm open to it. Just explain why.

Reply:
Because it is solver specific.

Reply:
> terminated at about

tries to let some uncertainity on the real termination time.
I would prefer to document that the used can expected the simulation termination before simulation stop time. May be the following text is more suitable:
_For current example settings, the termination condition should indeed be fulfiled right before  simulation stops._